test/torchaudio_unittest/datasets/dr_vctk_test.py

```diff
@@ -0,0 +1,135 @@
+from pathlib import Path
+
+from torchaudio.datasets import dr_vctk
+
+from torchaudio_unittest.common_utils import (
+    TempDirMixin,
+    TorchaudioTestCase,
+    get_whitenoise,
+    save_wav,
+)
+
+
+_SUBSETS = ["train", "test"]
+_CONDITIONS = ["clean", "device-recorded"]
+_SOURCES = ["DR-VCTK_Office1_ClosedWindow", "DR-VCTK_Office1_OpenedWindow"]
+_SPEAKER_IDS = range(226, 230)
+_CHANNEL_IDS = range(1, 6)
+
+
+def get_mock_dataset(root_dir):
+    """
+    root_dir: root directory of the mocked data
+    """
+    mocked_samples = {}
+
+    dataset_dir = Path(root_dir) / dr_vctk.FOLDER_IN_ARCHIVE
```

Comment:
Can you hard code `FOLDER_IN_ARCHIVE`? so that the test can detect the change if the library implementation accidentally changes it.